[5.3] Login Module show password

brianteeman opened the pull request:
### Summary of Changes
The login module password field has a button to reveal the password (change from *) however this button is not present if you have set the module to display labels. This PR fixes that.



### Testing Instructions
Create a login module for the site and set the option Display Labels to Text

![image](https://github.com/user-attachments/assets/ac0d486c-55c5-4ac4-b6fd-5efa91154a22)



### Actual result BEFORE applying this Pull Request

![image](https://github.com/user-attachments/assets/b04d7058-5400-4a6f-aef3-6dd1344c6706)



### Expected result AFTER applying this Pull Request
![image](https://github.com/user-attachments/assets/160807ca-f76a-40f6-8f2d-b0b12c0cc3ca)



### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

richard67 commented:
If it is a bug fix, why is it made for 5.4-dev and not for 5.3-dev? I'm ok with having it in 5.4, but it is still a while until we go stable with it, so people have to wait for that fix.

brianteeman committed:
[5.3] Login Module show password

The login module password field has a button to reveal the password (change from *)

Signed-off-by: BrianTeeman <brian@teeman.net>

richard67 committed:
Merge branch '5.3-dev' into login_module

ChristineWk commented:
I have tested this item :white_check_mark: successfully on d4f899add416bdc996681a75f8861f68bb0ae781<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/45626">issues.joomla.org/tracker/joomla-cms/45626</a>.</sub>

richard67 commented:
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/45626">issues.joomla.org/tracker/joomla-cms/45626</a>.</sub>

richard67 committed:
Merge branch '5.3-dev' into login_module

rdeutz commented:
Thanks

brianteeman commented:
thanks
